src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,118 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { SearchLink } from '../SearchLink/SearchLink';
+import { getSearchWith } from '../../utils/searchHelper';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <SearchLink
+          params={{
+            sex: null,
+          }}
+          className={classNames({ 'is-active': !sex })}
+        >
+          All
+        </SearchLink>
+
+        <SearchLink
+          params={{
+            sex: 'm',
+          }}
+          className={classNames({ 'is-active': sex === 'm' })}
+        >
+          Male
+        </SearchLink>
+
+        <SearchLink
+          params={{
+            sex: 'f',
+          }}
+          className={classNames({ 'is-active': sex === 'f' })}
+        >
+          Female
+        </SearchLink>
+      </p>
+
+      <div className="panel-block">
+        <p className="control has-icons-left">
+          <input
+            data-cy="NameFilter"
+            type="search"
+            className="input"
+            placeholder="Search"
+            value={query}
+            onChange={(event) => setSearchParams(
+              getSearchWith(searchParams, {
+                query: event.target.value || null,
+              }),
+            )}
+          />
+
+          <span className="icon is-left">
+            <i className="fas fa-search" aria-hidden="true" />
+          </span>
+        </p>
+      </div>
+
+      <div className="panel-block">
+        <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+          <div className="level-left">
+            {['16', '17', '18', '19', '20'].map(century => (
+              <SearchLink
+                params={{
+                  centuries: centuries.includes(century)
+                    ? centuries.filter(c => c !== century)
```

Comment:
Try to create another variable name, not `c`. Use one letter as variable name - is a bad practice.

Reply:
@dellachaise I have simple array here used only in one place and I compare element from centuries array with variable century. This is more than self-describle. But if you insist on this - I can change it. But I think it will have no practical meaning here.

Reply:
You could leave this way. In this case, usage could be different from project to project and in the future, you would do it according to the tools of your project.

Reply:
@dellachaise I understand it, of course.